### ISSUE TITLE
 Cannot read property 'stack' of null

### ISSUE THREAD

**Issue by haggen** [fgnass/node-dev#231]
```
/home/haggen/.../node_modules/node-dev/lib/wrap.js:38
  console.error(err.stack || err);
                    ^

TypeError: Cannot read property 'stack' of null
    at process.<anonymous> (/home/haggen/.../node_modules/node-dev/lib/wrap.js:38:21)
    at process.emit (events.js:315:20)
    at process._fatalException (internal/process/execution.js:165:25)
```

See [`wrap.js` line 38](https://github.com/fgnass/node-dev/blob/624665a57dedb70c4942abee1a83b48c301a5fc2/lib/wrap.js#L38).

Just found out my part of the issue: I was throwing `null` somewhere, as you've probably imagined. I suppose it still would be nice to check `err` before trying to access `err.stack`.

**Comment by bjornstar** [comment]
Published v5.1.0, you're still going to get an error on that line, but it will just say `uncaughtException null`, not much more I can do for you.

**Comment by haggen** [comment]
@bjornstar I couldn't ask for anything else! 🍻 Thank you! Have a nice one!
